experiments/neural_mcts/eval_scripts/plotting/make_plots.py

```diff
@@ -0,0 +1,64 @@
+import matplotlib.pyplot as plt
```

Comment:
Why not run `make_the_plot` from `old_make_plots.py` ?

Reply:
(just consider it, no need to write out a justification)

Reply:
at some point we will want to put them together. We need the functionality of both as we transition to better checkpoint naming (which is set up for new runs).